backend/db/db_objects/message_db_objects/Message.js

```diff
@@ -0,0 +1,8 @@
+class Message {
+  constructor(topicId, meesageAuthor, creationDate) {
+    this.topicId = topicId;
+    this.meesageAuthor = meesageAuthor;
```

Comment:
Always think how properties would look when you call them, message.meesageAuthor is much worse than message.author.